Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarStickyHeader.h"
+#import "BPKCalendarAppearance.h"
+#import <Backpack/Font.h>
+
+@interface FSCalendarStickyHeader (Private)
+
+@property (weak  , nonatomic) UIView  *contentView;
+@property (weak  , nonatomic) UIView  *bottomBorder;
+@property (weak  , nonatomic) FSCalendarWeekdayView *weekdayView;
+
+@end
+
+@implementation BPKCalendarStickyHeader
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        self.titleLabel.textAlignment = NSTextAlignmentLeft;
+        [self.weekdayView removeFromSuperview];
```

Comment:
This seems pretty hacky. Can you explain this code?

Reply:
We need to have the header containing the weekdays as an external header instead of having it as part of the section header.